src/parser/warrior/fury/modules/azerite/SimmeringRage.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber, formatPercentage, formatThousands } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import TraitStatisticBox from 'interface/others/TraitStatisticBox';
+import SPELLS from 'common/SPELLS/index';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import Events from 'parser/core/Events';
+
+
+const simmeringRageDamage = traits => Object.values(traits).reduce((obj, rank) => {
+  const [ damage ] = calculateAzeriteEffects(SPELLS.SIMMERING_RAGE.id, rank);
+  obj.damage += damage;
+  return damage;
+  }, {
+    damage: 0,
+  });
+
+class SimmeringRage extends Analyzer {
+  rampage = [SPELLS.RAMPAGE_1.id, SPELLS.RAMPAGE_2.id, SPELLS.RAMPAGE_3.id, SPELLS.RAMPAGE_4.id];
+  rampageDamage = 0;
+  simmeringDamage = 0;
+  damage = 0;
+  rageGen = 0;
+
+  constructor(...args) {
+    super(...args);
+
+    this.active = this.selectedCombatant.hasTrait(SPELLS.SIMMERING_RAGE.id);
+
+    const damage = simmeringRageDamage(this.selectedCombatant.traitsBySpellId[SPELLS.SIMMERING_RAGE.id]);
+    this.damage = damage;
+
+    this.addEventListener(Events.energize.by(SELECTED_PLAYER).to(SELECTED_PLAYER), this.onPlayerEnergize);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.onPlayerDamage);
+  }
+
+    onPlayerEnergize(event) {
+      if (event.ability.guid === 278841) {
+        this.rageGen += 1;
+      }
+    }
+
+    onPlayerDamage(event) {
+      if (!this.rampage.includes(event.ability.guid)) {
+        return;
+      }
+
+      this.simmeringDamage += this.damage;
```

Comment:
Unfortunately, it's slightly more complicated than this when calculating azerite trait damage
See: #2088 
See: https://github.com/WoWAnalyzer/WoWAnalyzer/pull/2749/files#diff-bd998464da96f154c459eeee0fd788c6R48
Unfortunately with rampage because it has 4 different abilities, it slightly complicates the calculations. The coefficients are here:
https://www.wowhead.com/spell=184367/rampage

Because this is a rather big change that will take some research I won't have this block the PR if you think it will take a while to implement

Reply:
Actually the correct syntax for this is here 
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/parser/warlock/demonology/modules/azerite/DemonicMeteor.js#L48